src/modules/ekf2/EKF/ekf.h

```diff
@@ -40,8 +40,7 @@
  *
  */
 
-#ifndef EKF_EKF_H
```

Comment:
This switched because of the "unity build" where ekf2 and the EKF backend are now built together as a single big compilation unit. This allows things like inlining across different translation units (cpp files).

Reply:
Thanks.  I'll revert it!

Reply:
Reverted and rebased against current master.